nri/index.html

```diff
@@ -1 +1,271 @@
 <!DOCTYPE html>
+<html lang="ja">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://unpkg.com/ress/dist/ress.min.css">
+  <link href="https://use.fontawesome.com/releases/v5.6.1/css/all.css" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+  <link rel="icon" href="./img/B4AC787D-9ED4-4DA8-8D46-88C35494D9EA_4_5005_c.jpeg">
+  <title>NRIの強み：ビジネスモデルの特徴 | 企業情報 | 野村総合研究所(NRI)</title>
+</head>
+
+<body>
+  <header>
+    <h1><a href=""><img src="./img/B4AC787D-9ED4-4DA8-8D46-88C35494D9EA_4_5005_c.jpeg" alt=""></a></h1>
+    <nav>
+      <ul class="header-nav">
+        <li><a href="">ナレッジ•インサイト</a></li>
+        <li><a href="">ソリューション•サービス</a></li>
+        <li><a href="">サステナビリティ</a></li>
+        <li><a href="">ニュース</a></li>
+        <li><a href="">企業情報</a></li>
+        <li><a href="">株式•投資家情報(IR)</a></li>
+        <li><a href="">採用情報</a></li>
+      </ul><a href=""></a>
```

Comment:
情報のないタグが存在しています。不要です。

```suggestion
      </ul>
```